src/1_resources.js

```diff
@@ -27,14 +27,17 @@ var _validator;
 /**
  * @param {boolean} required
  * @param {validationTypes|RegExp} type
+ * @param {string=} fallback
+ * @param {validationTypes|RegExp=} fallbackType
  * @throws {Error}
- * @return {_validator}
  */
-function validator(required, type) {
-	return function(endpoint, param, data) {
+function validator(required, type, fallback, fallbackType) {
```

Comment:
that way we don't need to modify all these resources, plus the cordova ones need to change too :P